### ISSUE TITLE
Unable to load postPublishHook: 'sentry-expo/upload-sourcemaps'. The module does not export a function.

### ISSUE THREAD

**Issue by spyshower** [expo/sentry-expo#61]
So once again a plethora of errors after an SDK update, but yea I am the bad guy. Out of nowhere, I got this:

```

Unable to load postPublishHook: 'sentry-expo/upload-sourcemaps'. The module does not export a function.
Please fix your postPublish hook configuration.
```
app.json:

```
{
  "expo": {
    "name": "Namee",
    "icon": "./assets/icons/app_icon.png",
    "description": "My app",
    "version": "0.5.18",
    "slug": "slug",
    "primaryColor": "#a649f2",
    "orientation": "portrait",
    "notification": {
      "icon": "./assets/icons/test2.png",
      "color": "#a609f2",
      "androidMode": "default",
      "androidCollapsedTitle": "Title"
    },
    "splash": {
      "backgroundColor": "#a649f2",
      "image": "./assets/splash/splash.png",
      "resizeMode": "contain"
    },
    "ios": {
      "icon": "./assets/icons/app_icon.png",
      "bundleIdentifier": "m.y.app",
      "buildNumber": "0.5.18",
      "supportsTablet": true
    },
    "android": {
      "versionCode": 18,

      "package": "m.y.app",
      "googleServicesFile": "./google-services.json",

      "permissions": [
        "CAMERA",
        "RECORD_AUDIO",
        "READ_EXTERNAL_STORAGE",
        "WRITE_EXTERNAL_STORAGE",
        "ACCESS_COARSE_LOCATION",
        "ACCESS_FINE_LOCATION"
      ]
    },
    "androidStatusBar": {
      "barStyle": "light-content",
      "backgroundColor": "#a640f2"
    },
    "sdkVersion": "33.0.0",
    "facebookScheme": "fb2075940980",
    "facebookAppId": "27594909294",
    "facebookDisplayName": "My app",
    "assetBundlePatterns": [
      "**/*"
    ],
    "hooks": {
      "postPublish": [
        {
          "file": "sentry-expo/upload-sourcemaps",
          "config": {
            "organization": "loox",
            "project": "loox",
            "authToken": "f77c77bc9d449c8b8a0f5ecae12"
          }
        }
      ]
    }
  }
}
```

**Comment by cooptwostar** [comment]
Probably don't want your authToken here. Looks like you did truncate it before posting though.
Is `node_modules/sentry-expo/upload-sourcemaps.js` present?

**Comment by spyshower** [comment]
I would like to say that sentry-expo causes SO much trouble without any reason. Like, it got fixed by me or by itself (don't remember/know), then it PREVENTED me from installing packages, now this error is back.

Actually, sentry-expo is gone from my node_modules but it exists in package.json. I thank god I am not a trash developer.

**Comment by cruzach** [comment]
@spyshower- really sorry you're having so much trouble with this module 😕 

What version are you using? Definitely weird that it's present in your `package.json` file but not in `node_modules`... Is that after running `npm (or) yarn install`?

**Comment by maksyukvadim** [comment]
`yarn install` - this works for me

**Comment by cruzach** [comment]
Closing this for inactivity from OP

**Comment by barberj** [comment]
I'm also having issues getting by the verification event step. I just noted that the logo by the ReactNative Platform is an apple icon. Does this only work for ios? I'm not running an ejected app. I've been following the instructions at https://docs.expo.io/versions/latest/guides/using-sentry. 

I'm running on an android simulator. I see the error in the emulator, but nothing in sentry. 

Thanks in advance.

**Comment by cruzach** [comment]
This works both for android and iOS, I'm guessing there's something off with the config for your project

**Comment by barberj** [comment]
> This works both for android and iOS, I'm guessing there's something off with the config for your project

Best channel to get advisement? New issue? Slack? Another comment here?

**Comment by cruzach** [comment]
Is this happening on a blank `expo init` project? I'd need more information about your project to try and help

**Comment by barberj** [comment]
It is not.

**Comment by fullhdpixel** [comment]
Had the same issue on sentry-expo@3.0.0-rc1 but its fixed on sentry-expo@3.0.0-rc2

**Comment by jer-sen** [comment]
I solved this issue with this config in my `app.json`:

  ```
  "hooks": {
      "postPublish": [
        {
          "file": "sentry-expo/dist/hooks/upload-sourcemaps",
          "config": {
            "organization": "...",
            "project": "...",
            "authToken": "..."
          }
        }
      ]
    }
  }
```

@cruzach the doc should be updated

**Comment by cruzach** [comment]
@jer-sen it's fixed in sentry-expo@3.0.0-rc2, as stated above
